packages/date-picker/src/DatePicker/DatePickerMenu/DatePickerMenuHeader/DatePickerMenuHeader.tsx

```diff
@@ -123,7 +123,9 @@ export const DatePickerMenuHeader = forwardRef<
       <div className={menuHeaderSelectContainerStyles}>
         <Select
           {...selectElementProps}
-          aria-label="Select month"
+          aria-label={`Select month - ${
+            monthOptions[month.getUTCMonth()].short
```

Comment:
I think we want `.long`. As-is this will announce "Select month - Dec selected".
Also, not sure how VO handles dashes like that

Reply:
i didn't hear any issues with the dash but I'll double check

Reply:
![Screenshot 2024-01-25 at 11 46 45 AM](https://github.com/mongodb/leafygreen-ui/assets/2414030/45b45e09-1ddc-4639-a304-749e50a6911a)

Do you know why it's saying "completion selected" ? 
Also, do you know why is says "group, (1 of 2)"? (the year select also says "1 of 2")

Reply:
Honestly, I'm not sure; there were a few issues I had while using VO, like not being able to navigate inside the selects. I was gonna investigate a little more and create tickets for them.